src/utils/updateTokenPrices.ts

```diff
@@ -103,3 +115,104 @@ async function getVeth2Price(
     return etherPrice
   }
 }
+
+export const getTokenPrice = (
+  tokens: BasicTokens,
+  dispatch: AppDispatch,
+  chainId: ChainId,
+): void => {
+  if (!tokens) return
+  const tokenAddresses = Object.keys(tokens)
+  const platform =
```

Comment:
I don't think we want to use `ethereum` as a default here. If we use the wrong platform for the tokens, then none of the addresses will match anyway, so we won't get any data from coingecko. This should probably log an error if platform is unknown.